socorro/database/cachedIdAccess.py

```diff
@@ -1,168 +0,0 @@
-# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
There's no evidence of `cachedIdAccess` in the codebase or configuration at all, so I removed this file.